haystack/utils/app_loading.py

```diff
@@ -14,6 +15,14 @@ def haystack_get_app_modules():
     """Return the Python module for each installed app"""
     return [i.module for i in apps.get_app_configs()]
 
+def haystack_get_search_indexes():
```

Comment:
I think I would move this to utils/loading.py since it's not really loading apps per-se. It would also keep the logic closer to where it's used.